### DISCUSSION TITLE
Option to hide stats/players on the right sidebar

### DISCUSSION THREAD

**Discussion by TomLewis** [gsand/mark2#143] [Ideas]
Could we get some way to toggle show/hide the players/stats on the right so we can get a full width console

**Comment by Column01** [comment]
Not sure this is easy to do since URWID (the stuff used to make the console window) is pretty baked in place. I could look into it though. Moving this to an issue as its a more appropriate section for feature requests

**Comment by Column01** [comment]
See: https://github.com/gsand/mark2/issues/145
